tests/commands/install_uninstall_test.py

```diff
@@ -278,7 +283,7 @@ def test_environment_not_sourced(tempdir_factory, store):
             },
             retcode=None,
         )
-        assert ret == 1
+        assert ret == 1, stderr
```

Comment:
this clobbers the pytest exception message

Reply:
This was left over debug that was accidentally committed.